style.css

```diff
@@ -0,0 +1,138 @@
+body {
+	width: 1440px;
+	height: 1024px;
+	margin: 0;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/evagame/pullRequest?prid=5788637)